maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/client/EvidenceApiClient.java

```diff
@@ -12,9 +12,9 @@
 @HttpExchange
 public interface EvidenceApiClient {
 
-  @PostExchange("/evidence")
+  @PostExchange()
   ApiCreateIncomeEvidenceResponse createEvidence(@RequestBody ApiCreateIncomeEvidenceRequest request);
 
-  @PutExchange("/evidence")
+  @PutExchange("")
```

Comment:
Can get rid of the brackets here - this needs tidying up in other Api clients too.